pkg/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -166,11 +167,15 @@ func getIgnitionConfig(assetsDir string, rtRepoURL string, nonIsolatedCpus *perf
 		return nil, err
 	}
 
+	enableRTKernel := profile.Spec.RealTimeKernel != nil &&
+		profile.Spec.RealTimeKernel.Enabled != nil &&
+		*profile.Spec.RealTimeKernel.Enabled
+
 	ignitionConfig.Systemd = igntypes.Systemd{
 		Units: []igntypes.Unit{
 			{
 				Contents: rtKernelService,
-				Enabled:  pointer.BoolPtr(true),
+				Enabled:  &enableRTKernel,
```

Comment:
Should we create at all RT kernel unit if the RT disabled, also pre-tunning also depends on the RT kernel unit, so I am interesting if it will work as expected?

Reply:
ah yes, good point
I'm wondering if we need the option to not install the RT kernel at all, when other performance features depend on it (no clue if all of them do?)

Reply:
other than that, pre-tuning uses `wants` and not `requires`, so it would be started without rt-kernel (to my best understanding)

Reply:
AFAIK it is correct, if an unit `wants` another unit and the latter can't be started, the former should run anyway (see: https://unix.stackexchange.com/questions/388586/systemd-requires-vs-wants - which in turn quotes manpages).
Question is however is if makes sense to do pretuning without rt-kernel, like Marc mentionedin on his grandparent comment.